packages/docmanager/src/manager.ts

```diff
@@ -110,6 +110,24 @@ class DocumentManager implements IDisposable {
     return this._activateRequested;
   }
 
+  /**
+   * Whether to autosave documents.
+   */
+  get autosave(): boolean {
+    return this._autosave;
+  }
+  set autosave(value: boolean) {
+    this._autosave = value;
+    each(toArray(this._contexts), context => {
```

Comment:
`this._contexts` is already an array, so you can just do `this._contexts.forEach(...)`.

This pattern in the file probably exists as a a holdover from an earlier version where it was an iterator and not an array. Would you mind changing this spot and the other `toArray(...)` places that do this?

Reply:
Good point. Changing now.